somsolet/models/client.py

```diff
@@ -48,6 +48,20 @@ class Client(models.Model):
         verbose_name=_('DNI'),
     )
 
+    def __str__(self):
+        return self.name
+
+
+class NotificationAddress(models.Model):
```

Comment:
What do you think if we add the arguments `related_name` and `related_query_name` to get client notification address more easily?

ref: https://docs.djangoproject.com/en/1.11/ref/models/fields/#django.db.models.ForeignKey.related_name

Reply:
Hello! :wave: 
We've added `related_name` in `client_file` and `notification_address` ForeignKeys in Project.
Is this what you meant?